### ISSUE TITLE
As a CIF Administrator, I want to track which projects are related to Methane Capture, so that I can track and report out on how many projects related to this type of activity

### ISSUE THREAD

**Issue by LindsayMacfarlane** [bcgov/cas-cif#1421]
#### Description:

Brief description of the work involved, including any user research context or key decisions.

#### Acceptance Criteria:

Given
When
Then

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

#### Notes:

- During usability testing, LP identified the potential need for adding Methane Capture to 'Project Type'.
- Is there an 'other' that can capture this? Will need to identify priority for this request 

#### Definition of Ready (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

#### Definition of Done (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

**Comment by Sepehr-Sobhani** [comment]
@LindsayMacfarlane Do we need to sort project types alphabetically too?! asking this because the newly added project type gonna show up after `Other` type in the drop-down.

**Comment by LindsayMacfarlane** [comment]
@nanyangpro - do you remember if there is a reason for the current order to the project type drop down?

@Sepehr-Sobhani - is it possible to keep the same order but put Methane Capture above 'other'?

**Comment by Sepehr-Sobhani** [comment]
@LindsayMacfarlane Yes It's possible, we need to add a new number field to the table and order project types based on that number but if we don't care about the order, we can just sort it alphabetically on the front-end.

**Comment by nanyangpro** [comment]
Hi @LindsayMacfarlane - There's no specific reason for the current ordering that I can recall. The implementation was simply following [the list of Project Types](https://bcgov.sharepoint.com/:w:/t/00584/EUEdqf-KoIFMtMtP3whIrYIBPpjZzFYh2c_nf6WMM5r58w?e=8Vaudf) the CIF tech team put together.

**Comment by Sepehr-Sobhani** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Climate-Action-Secretariat-60ca4121764d710011481ca2/issues/bcgov/cas-cif/1421?planning-poker) with Zenhub @BCerki @gurjmatharu @pbastia @dleard @tmastrom @sam-warren

**Comment by mikevespi** [comment]
We can also just handle the ordering in JS on the frontend rather than having the query do it. We already filter and sort the response, adding a `.sort()` to it is inexpensive on a list this small.
